tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -197,6 +197,26 @@ bool TypeTranslator::isScalarType(QualType type, QualType *scalarType) {
   return isScalar;
 }
 
+bool TypeTranslator::isRWByteAddressBuffer(QualType type) {
```

Comment:
Do we need two methods here? It seems these two are always called together, at least for now. I'd think combining them into one should be fine.